robot/client/client.go

```diff
@@ -1078,3 +1083,54 @@ func (rc *RobotClient) Version(ctx context.Context) (robot.VersionResponse, erro
 
 	return mVersion, nil
 }
+
+func addVersionMetadataToContext(ctx context.Context) context.Context {
+	info, _ := debug.ReadBuildInfo()
+	deps := make(map[string]*debug.Module, len(info.Deps))
+	for _, dep := range info.Deps {
+		deps[dep.Path] = dep
+	}
+	var apiVersion string
+	if dep, ok := deps["go.viam.com/api"]; ok {
+		apiVersion = dep.Version
+	}
```

Comment:
Building this map seems a bit excessive given that we only care about a single value in it. Can we just iterate through and look for `go.viam.com/api` rather than constructing the whole map?

Reply:
This map is also used for `go.viam.com/rdk`.

Reply:
Ahh, sure, but could we not also just update that value directly as well? Either way, it seems like we're storing a lot of information we don't actually care about.